hydroflow/examples/chat/main.rs

```diff
@@ -23,9 +37,7 @@ struct Opts {
     #[clap(value_enum, long)]
     role: Role,
     #[clap(long, value_parser = ipv4_resolve)]
-    addr: Option<SocketAddr>,
-    #[clap(long, value_parser = ipv4_resolve)]
-    server_addr: Option<SocketAddr>,
```

Comment:
Love this new simpler parameterization

Reply:
Created https://github.com/hydro-project/hydroflow/issues/1123 for follow-up.